storage/rocksdb/ha_rocksdb.cc

```diff
@@ -9933,15 +9932,15 @@ static void rdb_gen_normalized_tablename(const std::string *db,
 */
 int ha_rocksdb::create_table(const std::string &table_name,
                              const std::string &actual_user_table_name,
-                             const TABLE *table_arg,
+                             const TABLE &table_arg,
                              ulonglong auto_increment_value,
-                             dd::Table *table_def MY_ATTRIBUTE((__unused__)),
+                             const dd::Table &table_def [[maybe_unused]],
```

Comment:
This change will cause issue.. due to https://github.com/facebook/mysql-5.6/blob/fb-mysql-8.0.28/sql/sql_tmp_table.cc#L2279 will pass table_def as nullptr..

Reply:
Good catch. This can only happen with optimizer temp tables, which in future should be allowed to materialize in MyRocks too